kpconv_torch/datasets/NPM3D.py

```diff
@@ -29,17 +28,23 @@ def __init__(
         chosen_log=None,
         infered_file=None,
         output_dir=None,
-        split="training",
         use_potentials=True,
         load_data=True,
+        split="training",
     ):
+        PointCloudDataset.__init__(
+            self,
+            config=config,
+            datapath=datapath,
+            dataset="NPM3D",
+            chosen_log=chosen_log,
+            infered_file=infered_file,
+            output_dir=output_dir,
+            split=split,
+        )
         """
         This dataset is small enough to be stored in-memory, so load all point clouds here
         """
-        PointCloudDataset.__init__(self, "NPM3D")
-
-        self.train_save_path = get_train_save_path(output_dir, chosen_log)
-        self.test_save_path = get_test_save_path(infered_file, chosen_log)
```

Comment:
Same remarks:

- do not add code before docstring
- use `super().__init__(...)` to call the constructor of the parent class